site/src/pages/tutorials/thrift/blog/create-service.mdx

```diff
@@ -0,0 +1,82 @@
+---
+menuTitle: "Create a service"
+order: 1
+category: thrift
+type: step
+targetLang: java
+---
+
+# Creating a service
+
+As the first step of the tutorial, we'll create a simple service with a dummy method in Thrift and implement the service in Java.
+
+<TutorialSteps current={1} />
+
+## What you need
+
+No preparation is required for this step. Do check that you've prepared the [prerequisites](/tutorials/thrift/blog/#prerequisites).
+
+## 1. Create a thrift file
+
+Create a thrift file, `blog.thrift` in the `{project_root}/src/main/thrift` folder as follows.
+In the thrift file, let's define the `BlogService` service with the `hello()` method.
+
+```thrift filename=blog.thrift
+namespace java example.armeria.blog.thrift
+
+/**
+ * My first BlogService in Armeria!
+ */
+service BlogService {
+    string hello()
+}
+```
+
+<Tip>
+
+See [Sample service structure](/tutorials/thrift/blog#sample-service-structure) for the overall folder structure.
+
+</Tip>
+
+## 2. Compile the thrift file
+
+Compile the `blog.thrift` file to generate Java code.
+You can refer to the full [build.gradle](https://github.com/line/armeria-examples/tree/main/tutorials/thrift/build.gradle) file for generating code with [Gradle Thrift Plugin](https://github.com/jruyi/thrift-gradle-plugin).
+
+```bash
+./gradlew build
+```
+
+You'll see the generated Java code in the `{project_root}/build/generated-sources/thrift/gen-java/example/armeria/blog/thrift/` folder.
+
+## 3. Implement the service
+
+Now, let's implement the service in Java.
+
+1. Create a file, `BlogServiceImpl.java`.
+2. Declare the `BlogServiceImpl` class implementing the `BlogService` service we defined earlier in Thrift.
```

Comment:
It would be beneficial to provide additional information.
Let me add this documentation link as well :)

> See [Running a Thrift service](https://armeria.dev/docs/server-thrift) for more information.

Reply:
Also, wouldn't it be better to place the tip box right above **What's next**, at the end of **3. Implement the service**?
I think the tip box draws too much attention and might interrupt users following the steps.

Reply:
> Also, wouldn't it be better to place the tip box right above What's next, at the end of 3. Implement the service?
> I think the tip box draws too much attention and might interrupt users following the steps.

Sounds good to me 👍